### ISSUE TITLE
Julia 1.8.5 breaks reading Excel worksheet data to dataframe 

### ISSUE THREAD

**Issue by pubmania** [felipenoris/XLSX.jl#221]
After I have updated to Julia 1.8.5, I found that 

```
df = DataFrame(XLSX.readtable("myexcel.xlsx", "sheet1")) 
```
gave the following error:

**_ArgumentError: 'Tuple{Vector{Any}, Vector{Symbol}}' iterates 'Vector{Any}' values, which doesn't satisfy the Tables.jl `AbstractRow` interface_**

When I changed the code to following, it worked just fine:

```
df = DataFrame(XLSX.readtable("myexcel.xlsx", "sheet1")...)
```

Looks like reverse of what is mentioned on the [Migration Guide](https://felipenoris.github.io/XLSX.jl/stable/migration/#Reading-DataFrames) now is working so whatever was introduced in V0.8 seems to have been impacted by upgrading to Julia 1.8.5

**Comment by felipenoris** [comment]
@pubmania many thanks for your report! This looks bad.

**Comment by felipenoris** [comment]
@pubmania Hi! I couldn't reproduce your problem. It is most likely that you're using an older version of XLSX on Julia v1.8.5. Please check your XLSX version.

```julia
julia> DataFrame(XLSX.readtable("Pasta1.xlsx", 1))
3×3 DataFrame
 Row │ COL1  COL2  COL3 
     │ Any   Any   Any  
─────┼──────────────────
   1 │ a     1     hey
   2 │ b     2     you
   3 │ c     3     out

julia> DataFrame(XLSX.readtable("Pasta1.xlsx", "Plan1"))
3×3 DataFrame
 Row │ COL1  COL2  COL3 
     │ Any   Any   Any  
─────┼──────────────────
   1 │ a     1     hey
   2 │ b     2     you
   3 │ c     3     out

julia> VERSION
v"1.8.5"

(@v1.8) pkg> st
Status `~/.julia/environments/v1.8/Project.toml`
  [a93c6f00] DataFrames v1.4.4
  [fdbf4ff8] XLSX v0.8.4
```

**Comment by pubmania** [comment]
Yes you are right.. am sorry I just assumed that as it was working earlier and is not working now upgrade was an issue. Apparently at some point package ExcelFiles was installed which restricts XLSX to not go past 0.74 hence the issue.
